microsoftgraph/microsoft-graph-explorer-v4
# Fix: Consent button color

## Overview
Fixes #1672 

### Demo
Light mode
![image](https://user-images.githubusercontent.com/45680252/164453431-daf92ab5-4db9-465e-ad8c-3076290553f1.png)
(disabled )

![image](https://user-images.githubusercontent.com/45680252/164453497-43e0fc1f-39c0-45a3-b9f2-6ab524417de6.png)
 (enabled)

High contrast mode
![image](https://user-images.githubusercontent.com/45680252/164453582-a1089814-0f84-47e9-bee9-36cb6dda0aac.png)
 (disabled )
![image](https://user-images.githubusercontent.com/45680252/164453695-0aec5d78-a0dc-414e-a976-e4c404c9e43c.png)
(enabled)

Dark Mode
![image](https://user-images.githubusercontent.com/45680252/164453764-6a316e88-1fed-4806-86c8-0baac8d3856c.png)
(disabled)
![image](https://user-images.githubusercontent.com/45680252/164453825-7a155366-e200-45ae-ad2c-b753b47ebb2b.png)
(enabled)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch
Sign in with any account
Click on 'More Actions' button and change theme to light mode
Open permission panel and notice the color on the Consent button. Select a few permissions' checkboxes. Notice the color change

Repeat the same for other color modes
